multi-agent/lib/mas/blueprints/models/prompt_shortcuts.py

```
@@ -0,0 +1,132 @@
+
+    id: str = Field(default_factory=_short_id)
+    kind: PromptShortcutKind = PromptShortcutKind.MANUAL
+    text: str
+
+    _HEX8_RE = re.compile(r"^[0-9a-f]{8}$")
+
+    @field_validator("id")
+    @classmethod
+    def validate_id(cls, v: str) -> str:
+        if not cls._HEX8_RE.match(v):
+            raise ValueError("id must be exactly 8 lowercase hex characters")
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)